sdks/corda/src/main/kotlin/com/weaver/corda/sdk/AssetManager.kt

```diff
@@ -104,6 +107,78 @@ class AssetManager {
             }
         }
 
+        @JvmStatic
+        @JvmOverloads fun createFungibleAssetPledge(
```

Comment:
One more thing is, I'd suggest to encapsulate Asset Transfer functions in SDK, in separate class than HTLC.
May be `AssetTransferManager`, then we can have simple names like `PledgeAsset`, `ClaimAsset`, `ReclaimAsset`, (rather than ClaimPledgedAsset which seems long) since class are different, there won't be collision with HTLC function names.